spec/deaf_grandma_spec.rb

```diff
@@ -9,10 +9,10 @@
   end
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA!!!")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    expect(script.speak("BYE BYE Grandma! BYE!")). to eq "SEE YOU LATER SONNY!"
```

Comment:
I think the idea here is supposed to be that the user inputs 'BYE' three *separate* times to generate that response.  That's why it's a little bit hard to test.